reanalysis/utils.py

```diff
@@ -642,7 +641,7 @@ def check_read_depth(self, sample: str) -> list[str]:
         Returns:
             return a flag if this sample has low read depth
         """
-        threshold = get_config()['filter'].get('minimum_depth', 10)
+        threshold = SCRIPT_CONFIG['filter'].get('minimum_depth', 10)
```

Comment:
Not relevant as part of this change, but you could assert filter in the config when you grab it, to save getting to this point without the key